LCLSGeneral/LCLSGeneral/POUs/Hardware/FB_GetPLCHostname.TcPOU

```diff
@@ -0,0 +1,45 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_GetPLCHostname" Id="{bd570dd7-dba7-4a18-a0ee-a97d90c7deaf}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_GetPLCHostname
+VAR_INPUT
+    fRetryDelay		: TIME := T#10s;
+    bExecute		: BOOL := TRUE;
```

Comment:
A few non-blocking general coding conventions questions about these two lines:
- for variable prefixing, do we normally use `f` or `t` for the `TIME` type? (if we're normally using `t`, then that makes `tRetry` below confusing, if we're normally using `f` then that makes it confusing that I can't just pass 10.3 to this function block)
- is it normal to default `bExecute` to TRUE?
- is it normal to use `bExecute` for things that aren't rising edge activations? I've also seen `bEnable` floating around.

Reply:
I don't have any good answers to those convention questions - but I'm certainly open for input. It would be nice if we came up with our own set of PLC coding style guide some day. I'm definitely not qualified to write those at this point, though.

Prefix reference - https://infosys.beckhoff.com/english.php?content=../content/1033/tcplclibsoverview/html/tcplclibsprogrammingconventions.htm

Reply:
I changed `f` to `t`, made `bExecute` not default to anything. Not sure about the edge-triggered nature of it (or if it should be named `bEnable` or what)

Reply:
I've added that page to my bookmarks :+1: very helpful.
I was hoping Maggie or Alex would want to comment on the third point since they've done a lot of this stuff

Reply:
https://infosys.beckhoff.com/content/1033/tc3_plc_intro/3363785099.html?id=3168322580474451815

Prefix reference in TC3 space.

Reply:
On the third point.... 
In 5 minutes of skimming the documentation I found bExecute always as a rising edge input to activate the FB functionality. I found bEnable used as a "press and hold to run" input.

bExecute:
https://infosys.beckhoff.com/content/1033/tf6310_tc3_tcpip/84149131.html?id=4003306490822847246
bEnable:
https://infosys.beckhoff.com/content/1033/tf6225_tc3_ethercat_external_sync/3623206539.html?id=3422692123219913233

I could almost swear I saw somewhere in the docs once upon a time a description of how bExecute is used in Beckhoff FB (or maybe it was PLCOpen standards), as always a rising edge start.